install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -0,0 +1,155 @@
+# Migration Guide from Thanos to Kubecost 2.0 (Aggregator)
+
+This tutorial is intended to help our users migrate from the legacy Thanos federation architecture to [Kubecost 2.0 (Aggregator)](aggregator.md). There are a few requirements in order to successfully migrate to Kubecost 2.0. This new version of Kubecost includes a new backend Aggregator which handles the ETL data built from source metrics more efficiently. Kubecost 2.0 will provide new features, optimize the Kubecost's UI performance, and enhance the user experience.
+
+## Key changes
+
+* Assets and Allocations are now paginated
+   * Frontend/API users never have full set of information
+   * Use `offset`/`limit` parameters to govern pagination
+* New data available for querying every 2 hours (can be adjusted)
+* Embedded DuckDB database serves queries
+   * Data no longer queried directly from bingen files
+   * Substantial query speed improvements even when pagination not in effect
```

Comment:
This is all correct, but I don't think it should matter to a user that we have an embedded DuckDB database, all they should care about is the last bullet "substantial speed improvements".

Reply:
@michaelmdresser - thanks for chiming in here. Most current customers I've been discussing 2.0 with have been asking for details about the architectural changes, breaking changes, etc. I don't think this needs to go in our formal 2.0 doc(s) but I think it'd be useful here. Perhaps we don't need to mention its a DuckDB database specifically? Maybe it's not relevant enough?

Reply:
I'd rather not mention what database -- databases are a touchy topic for many Kubernetes users. The fact that we're using an embedded database mitigates many of those concerns, but I don't want to have to get into the weeds to explain it. I'd go for something more like "overhauled query execution engine which efficiently combines multi-cluster data and serves it to queries faster than ever before". That wording could use some work, but that's the idea we want to convey. Then we can just lean on decent architecture diagrams (more to do there!) to show the high-level changes.

Reply:
@thomasvn lgtm. Thanks!

Reply:
LGTM, thanks Thomas